### ISSUE TITLE
Staging Review finding: Unnecessary HTML structure in "Your appointment" section after check-in

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#89004]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Modernized Check-in
**Product:** Pre-Check-in & eCheck-in
**Feature:** New Landing page design

## Findings details
**VA.gov Experience Standard - issue:** [User can't easily perform the primary task free of distractions.](https://design.va.gov/about/experience-standards/#user-can-perform-the-primary-task-free-of-distractions)
**VA.gov Experience Standard - category:** [Efficiency](https://design.va.gov/about/experience-standards/#category-efficiency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On /appointment-check-in/complete/[id], the "Your appointment" section displays the date, time, type, etc of the appointment you just checked in for. That chunk of content includes list semantics (`ul` + `li`) and an H3 heading.

Lists and headings are great for helping users navigate content that consists of many parts, and that structural information is announced to screen reader users. But if I'm understanding the flow here, this section will never have more than one list item, and will never have appointments at other times that need to be differentiated from --- right? You can only check in for one appointment at a time, so the page you land on when completing your check-in will only have one appointment at a time.

If that's a correct understanding, then the list and heading semantics aren't really necessary here and may be distracting to screen reader users.

<details><summary>Link, screenshot or steps to recreate</summary>

![Heading text reading "Your appointment" followed by Monday, July 22, 2024, a heading reading 4:30pm, followed by VA appointment, In person at VISTA-API STAGING, Clinic: SLEEP CLINIC, and a Details link.](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/working-with-vsp/vsp-collaboration-cycle/staging-review-images/Brian-53488-2.png?raw=true)

</details>

## Recommended action

Assuming you can only ever have one appointment listed here, I think you're safe to just make this text. No `ul`/`li`, no H3 required.

If it is possible for multiple appointments to be listed here (or if that's on the roadmap for a future trip through the Collab Cycle), please disregard!

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by loripusey** [comment]
Please [add your planning poker estimate](https://app.zenhub.com/workspaces/Check-in-Experience-61fc23a2cb8a14001132e102/issues/department-of-veterans-affairs/va.gov-team/89004?planning-poker) with Zenhub @brianseek

**Comment by brianseek** [comment]
@benbrasso-agile6, @zach-park, @cgednalske this is ready to validate. See screenshot:
<img width="2000" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBMlZxQmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--880a0a139fda2bd617b238b7d9aa95a4c80a4590/no-li.png" alt="no-li.png" />

**Comment by benbrasso-agile6** [comment]
I suppose this looks good; doesn't look like I see any veteran facing changes (outside the implementation)

**Comment by loripusey** [comment]
Let's merge @brianseek if it hasn't been already, closing this ticket.

**Comment by brianseek** [comment]
merged it was already
